tfhe/src/shortint/ciphertext/standard.rs

```diff
@@ -138,7 +138,13 @@ impl Ciphertext {
         self.noise_level
     }
 
-    pub fn set_noise_level(&mut self, noise_level: NoiseLevel) {
+    #[cfg_attr(any(feature = "noise-asserts", test), track_caller)]
```

Comment:
Could be a simple `#[track_caller]` to avoid some complexity with features

Reply:
Yep went a bit too far here 😁